shell/ev-window.c

```diff
@@ -2190,21 +2190,19 @@ set_uri_mtime (GFile        *source,
 	       EvWindow     *ev_window)
 {
 	GFileInfo *info;
-	GError *error = NULL;
-
-	info = g_file_query_info_finish (source, async_result, &error);
-
-	if (error) {
-		ev_window->priv->uri_mtime = 0;
-		g_error_free (error);
-	} else {
-		GTimeVal mtime;
-
-		g_file_info_get_modification_time (info, &mtime);
-		ev_window->priv->uri_mtime = mtime.tv_sec;
+	gint      utime = -1;
```

Comment:
Is it correct? It's `gint64` in the similar `g_date_time_to_unix` call below...

Reply:
@monsta  You're right. It's `gint64`.
 
https://github.com/mate-desktop/atril/blob/992c27c2c8c864ec203d1ed25f3f678c730e631e/shell/ev-window.c#L202